bosh-release/README.md

```diff
@@ -1,127 +1,43 @@
-    -v outbound_network_name=NatNetwork
-    ```
-
-4. Once VM with BOSH Director is running, point your CLI to it, saving the environment with the alias vbox:
-
-    ```
-    bosh -e 192.168.50.6 alias-env vbox --ca-cert <(bosh int vbox/creds.yml --path /director_ssl/ca)
-    ```
-
-5. Obtain generated password to BOSH Director:
-
-    ```
-    bosh int vbox/creds.yml --path /admin_password
-    ```
-
-6. Log in using admin username and generated password:
-
-    ```
-    bosh -e vbox login
-    ```
-
-7. Download Oracle JDK 1.8 from Oracle website and WSO2 Identity Server 5.3.0 via WSO2 Update Manager (WUM).
+## Prerequisites
 
-8. Add above distributions as blobs:
+1. Linux/Unix environment
+2. Install [BOSH Lite](https://bosh.io/docs/bosh-lite.html#install)
+3. Install [Docker](https://docs.docker.com/engine/installation/)
+4. (Optional)Install [WSO2 Update Manager](http://wso2.com/wum)
 
-    ```
-    bosh -e vbox add-blob jdk-8u144-linux-x64.tar.gz oraclejdk/jdk-8u144-linux-x64.tar.gz
-    bosh -e vbox add-blob wso2is-5.3.0.zip wso2is/wso2is-5.3.0.zip
-    bosh -e vbox -n upload-blobs
-    ```
+## Quick Start Guide
 
-9. Create the WSO2 Identity Server bosh release:
+1. Create a workspace directory
+   ```
+   mkdir wso2-is-bosh-workspace
+   ```
 
+2. Navigate to the workspace directory and clone this repository
     ```
-    bosh -e vbox create-release --force
+    cd wso2-is-bosh-workspace
+    git clone https://github.com/wso2/pivotal-cf-is --depth 1
     ```
 
-10. Upload the WSO2 Identity Server bosh release to BOSH Director:
+3. Download JDK jdk-8u144-linux-x64.tar.gz binary and copy that to the workspace directory
 
-    ```
-    bosh -e vbox upload-release
-    ```
+4. Download MySQL JDBC driver mysql-connector-java-5.1.44-bin.jar binary and copy that to the workspace directory
 
-11. Download latest bosh-lite warden stemcell from bosh.io and upload it to BOSH Director:
-    
-    ```
-    wget https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-    bosh -e vbox upload-stemcell bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-    ```
+5. Get the WSO2 Identity Server 5.3.0 WUM updated distribution or the released [distribution](https://wso2.com/identity-and-access-management) and copy that to the workspace directory.
 
-12. Deploy the WSO2 Identity Server bosh release manifest in BOSH Director:
-
-    ```
-    bosh -e vbox -d wso2is deploy wso2is-manifest.yml
-    ```
-
-13. Add route to VirtualBox network:
-
-    ```
-    sudo route add -net 10.244.0.0/16 192.168.50.6 # Mac OS X
-    sudo route add -net 10.244.0.0/16 gw 192.168.50.6 # Linux
-    route add 10.244.0.0/16 192.168.50.6 # Windows
-    ```
-
-14. Find the VM IP address via the bosh CLI and access the WSO2 Identity Server Store via a web browser:
+6. Give execute permissions to the wso2-is-bosh-release\setup.sh file within your worspace directory and execute
```

Comment:
Why does the user needs to update the execute permission of the setup.sh file?

Reply:
Updated the file permission. This instruction will no longer be needed. Will update